src/app/services/search/search.service.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { SearchService } from './search.service';
+
+describe('SearchService', () => {
+  let httpMock: HttpTestingController;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+    });
+    httpMock = TestBed.get(HttpTestingController);
+  });
+
+  it('should be created', () => {
+    const service: SearchService = TestBed.get(SearchService);
+    expect(service).toBeTruthy();
+  });
+
+  it('should use search service', () => {
+    const service: SearchService = TestBed.get(SearchService);
+    const mockUrl = "https://landville-backend-web-api.herokuapp.com/api/v1/properties/?"
```

Comment:
Hello @JamesMudidi , we can do away with the `actual backend` here since we won't be making any actual network requests

Reply:
Done